### ISSUE TITLE
having blockquotes styled in the color of comments is not ideal

### ISSUE THREAD

**Issue by alerque** [vim-pandoc/vim-pandoc-syntax#287]
Having block quotes styled in the syntax color of comments is somewhat less than ideal. Admittedly comments are not native to Markdown, but Pandoc does support HTML style comments (in the AST as raw inline html) by default and they often get used that way. I use it along with comment plugins as part of copy editing workflow.

I don't know if there are any colors left to use, but block quotes looking a little less like they are something being left out of the party would be nice.

**Comment by fmoralesc** [comment]
I agree... Actually, we could create slightly less saturated versions of the normal text highlighting using some of the functions from https://github.com/vim-pandoc/vim-pandoc-syntax/blob/master/autoload/pandoc/syntax/color.vim: we can retrieve the colors of syntax elements, and desaturate them.

**Comment by fmoralesc** [comment]
That code has been there waiting for its moment to shine for almost 5 years now :p

**Comment by bpj** [comment]
What about using the String highlight group? Comment is clearly wrong.

Den tors 16 maj 2019 22:18Felipe Morales <notifications@github.com> skrev:

**Comment by bpj** [comment]
I have been highlighting both blockquotes and notes as `Special` for a while. Works well for me.

**Comment by fmoralesc** [comment]
The minimized syntax flavor from #331 actually drops all highlighting for block quotes (it's kept only as a hint for autoformatting).
